powerhome/playbook
# [PLAY-610] Converting Loading Inline kit to Typescrit

#### Breaking Changes

No

#### Runway Ticket URL

[Runway Ticke](https://nitro.powerhrg.com/runway/backlog_items/PLAY-610)

#### How to test this

Use the Loading Inline kit and ensure it works as expected after Typescript conversion

#### Checklist:
